tests/performance_tests/read_log_data.py

```diff
@@ -8,6 +8,7 @@
 parser.add_argument("--start_block", type=int, help="First significant block number in the log", default=2)
 parser.add_argument("--cease_block", type=int, help="Last significant block number in the log")
 parser.add_argument("--num-blocks-to-prune", type=int, default=2, help="The number of potentially non-empty blocks, in addition to leading and trailing size 0 blocks, to prune from the beginning and end of the range of blocks of interest for evaluation.")
+parser.add_argument("--save-json", type=bool, default=False, help="Whether to save stats as json")
```

Comment:
Should a path argument be provided to dictate where the json file should be written/saved?  If so, would probably be good here and in `performance_test_basic.py`. Could provide a sane default for if only `--save-json` is specified.